src/Matrix.jl

```diff
@@ -91,6 +91,20 @@ check_square(S::MatAlgebra) = S
 #
 ###############################################################################
 
+@doc Markdown.doc"""
+    nrows(a::MatSpace)
+
+Return the number of rows of the given matrix space.
+"""
+nrows(a::MatSpace) = a.nrows
+
+@doc Markdown.doc"""
+    ncols(a::MatSpace)
+
+Return the number of columns of the given matrix space.
+"""
+ncols(a::MatSpace) = a.ncols
```

Comment:
Technically, these should be in `generic/`, right?

Reply:
I put them there originally and then realised that we may as well make them fallback defaults for people who have an nrows/ncols field in their matrix space object.